xblockutils/public/studio_container.js

```diff
@@ -0,0 +1,55 @@
+function StudioContainerXBlockWithNestedXBlocksMixin(runtime, element) {
+    var $buttons = $(".add-xblock-component-button", element);
```

Comment:
Nit: I generally prefer it if we have a variable like `var $element = $(element)` and then use `$element.find()` for selection because it's easier to avoid mistakes where the qualifier is omitted.